wsgi/protocolhttp.cpp

```diff
@@ -326,6 +326,17 @@ void ProtocolHttp::parseHeader(const char *ptr, const char *end, Socket *sock) c
     } else if (!protoRequest->headerHost && key == QLatin1String("HOST")) {
         protoRequest->serverAddress = value;
         protoRequest->headerHost = true;
+    } else if (usingFrontendProxy && !protoRequest->X_Forwarded_For && key == QLatin1String("X_FORWARDED_FOR")) {
```

Comment:
I think for readability you shoul only have an else if (usingFrontendProxy) { then the ifs...

Reply:
in the works...